x-pack/plugins/security_solution/public/management/cypress/tasks/run_endpoint_loader.ts

```diff
@@ -12,7 +12,7 @@ export const runEndpointLoaderScript = () => {
     hostname: HOSTNAME,
     configport: PORT,
   } = Cypress.env();
-  const script = `node scripts/endpoint/resolver_generator.js --node="http://${ELASTICSEARCH_USERNAME}:${ELASTICSEARCH_PASSWORD}@${HOSTNAME}:9220" --kibana="http://${ELASTICSEARCH_USERNAME}:${ELASTICSEARCH_PASSWORD}@${HOSTNAME}:${PORT}" --delete --numHosts=1 --numDocs=1 --fleet --withNewUser=santaEndpoint:changeme`;
+  const script = `node scripts/endpoint/resolver_generator.js --node="http://${ELASTICSEARCH_USERNAME}:${ELASTICSEARCH_PASSWORD}@${HOSTNAME}:9220" --kibana="http://${ELASTICSEARCH_USERNAME}:${ELASTICSEARCH_PASSWORD}@${HOSTNAME}:${PORT}" --delete --numHosts=1 --numDocs=1 --fleet --withNewUser=santaEndpoint:changeme --anc=1 --gen=1 --ch=1 --related=1 --relAlerts=1`;
```

Comment:
are the new arguments here to make the generator faster?

Reply:
Exactly, the goal of these changes is create the minimum data needed for the existing tests